src/lib/functions/schedule.js

```diff
@@ -0,0 +1,24 @@
+const { NETLIFYDEVERR, NETLIFYDEVLOG } = require('../../utils/logo')
```

Comment:
[dust] Should this file be named `scheduled.js`?

Reply:
aggreed, 83924b9